x/evm/keeper/abci.go

```diff
@@ -35,8 +33,6 @@ func (k *Keeper) BeginBlock(ctx sdk.Context) error {
 // KVStore. The EVM end block logic doesn't update the validator set, thus it returns
 // an empty slice.
 func (k *Keeper) EndBlock(ctx sdk.Context) error {
```

Comment:
do we still need [reset](https://github.com/crypto-org-chain/ethermint/blob/a5b1f1c44016976bb14da255ac7424d3f1199323/x/evm/keeper/state_transition.go#L245-L251) the gas meter?

Reply:
that one is needed, that's how the cosmos tx calculate the gas used